paprika/tests/test_tleap.py

```diff
@@ -392,3 +392,76 @@ def test_multiple_pdb_files(clean_files):
     assert "b = loadpdb cb6-but-minimized.pdb" in lines
     assert "combine" in lines
     assert "savepdb model multi.pdb" in lines
+
+
+def test_conversions(clean_files):
+    """ Test that conversion methods work in Tleap module. """
+    temporary_directory = os.path.join(os.path.dirname(__file__), "tmp")
+
+    but_frcmod = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "../data/cb6-but/but.frcmod")
+    )
+    but_mol2 = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), "../data/cb6-but/but.mol2")
+    )
+
+    sys = System()
+    sys.template_lines = [
+        "source leaprc.gaff",
+        f"loadamberparams {but_frcmod}",
+        f"BUT = loadmol2 {but_mol2}",
+        f"model = loadmol2 {but_mol2}",
+    ]
+    sys.output_path = temporary_directory
+    sys.output_prefix = "but"
+    sys.pbc_type = None
+    sys.neutralize = False
+    sys.build()
+
+    from paprika.utils import is_file_and_not_empty
+
+    # Gromacs ParmEd test
+    sys.convert_to_gromacs(overwrite=True)
+    top_file = os.path.join(sys.output_path, sys.output_prefix + ".top")
+    gro_file = os.path.join(sys.output_path, sys.output_prefix + ".gro")
+    assert is_file_and_not_empty(top_file) is True
+    assert is_file_and_not_empty(gro_file) is True
+
+    # Gromacs InterMol test
+    sys.convert_to_gromacs(toolkit=System.ConversionToolkit.InterMol)
+    assert is_file_and_not_empty(top_file) is True
+    assert is_file_and_not_empty(gro_file) is True
+    top_file = os.path.join(sys.output_path, sys.output_prefix + "_from_amber.top")
+    gro_file = os.path.join(sys.output_path, sys.output_prefix + "_from_amber.gro")
+    assert is_file_and_not_empty(top_file) is True
+    assert is_file_and_not_empty(gro_file) is True
```

Comment:
Totally out of scope here, but if for any reason you find yourself needing to also compare the energies before and after conversion, InterMol does that pretty well. I'll be working on doing this independently elsewhere in the next month or two, so it could possibly be something you can get for free in the future.

Reply:
Yeah, that would be a better way for testing.

Reply:
Yes, but they're mostly/all conda installable (save Desmond, I think) so it's possible to do that in a separate CI build

Reply:
I will (most) probably need help in configuring the CI build (sometime in the future). This will be useful to test the simulation wrappers as well.